test/fixtures/mappers.yml

```diff
@@ -27,6 +27,15 @@ core_collectionobject_6_0:
   url: https://raw.githubusercontent.com/collectionspace/cspace-config-untangler/master/data/mappers/core_6_0_0-collectionobject.json
   status: true
 
+core_nonhierarchicalrelationship_6_1:
+  title: nonhierarchicalrelationship (core-6-1-0)
+  profile: core
+  type: nonhierarchicalrelationship
+  digest: abc3
```

Comment:
@kspurgin the digest needs to be unique for the mappers to be valid:

```ruby
[6] pry(#<PreprocessJobTest>)> Mapper.where(title: 'collectionobject (core-6-0-0)').first.valid?
=> false
[7] pry(#<PreprocessJobTest>)> Mapper.where(title: 'nonhierarchicalrelationship (core-6-1-0)').first.valid?
=> false
```

Updating the digest allows the tests to pass.